src/config/config.go

```diff
@@ -0,0 +1,68 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+type Configuration struct {
+	//cast onto the needed type when processing in algos
```

Comment:
can you cast into more complex types rather than relying on other code to do it? e.g. https://go.dev/play/p/uk3yQO1HysM

Reply:
will change once it is more clear exactly where/how the configuration parameters will be used ->  some time during this sprint